CMakeLists.txt

```diff
@@ -49,14 +49,17 @@ if(MSVC AND (MSVC_VERSION LESS 1600))
 endif(MSVC AND (MSVC_VERSION LESS 1600))
 
 # openmp
-find_package(OpenMP)
-if (OPENMP_FOUND)
-  add_definitions(-fopenmp -DHAVE_OPENMP)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
-  if (CMAKE_COMPILER_IS_GNUCC)
-    set(EXTRA_LIBS ${EXTRA_LIBS} gomp)
-  endif(CMAKE_COMPILER_IS_GNUCC)
-endif(OPENMP_FOUND)
+set(USE_OPEN_MP "true" CACHE BOOL "Set to OFF to not use OpenMP")
+if(USE_OPEN_MP)
+	find_package(OpenMP)
```

Comment:
please make the indentation consistent with the rest of the file. (2 space indent)